core/players-handbook-2024/backgrounds/background-acolyte.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Acolyte</name>
+		<description>Acolyte Background</description>
+		<author url="http://dnd.wizards.com/products/tabletop-games/rpg-products/rpg_playershandbook">Wizards of the Coast</author>
+		<update version="0.0.2">
+			<file name="background-acolyte.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/core/players-handbook-2024/backgrounds/background-acolyte.xml" />
+		</update>
+	</info>
+
+	<element name="Acolyte" type="Background" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_BACKGROUND_ACOLYTE">
+		<description>
+			<table>
+				<tr><td><b>Ability Scores:</b></td><td> Intelligence, Wisdom, Charisma</td></tr>
+				<tr><td><b>Feat:</b> </td><td> Magic Initiate (Cleric) (see chapter 5)</td></tr>
+				<tr><td><b>Skill Proficiencies:</b></td><td> Insight and Religion</td></tr>
+				<tr><td><b>Tool Proficiency:</b> Calligrapher’s Supplies</td><td> Calligrapher’s Supplies</td></tr>
+				<tr><td><b>Equipment:</b></td><td> <i>Choose A or B:</i> (A) Calligrapher’s Supplies, Book (prayers), Holy Symbol, Parchment (10 sheets), Robe, 8 GP; or (B) 50 GP</td></tr>
+			</table>
+			<p>You devoted yourself to service in a temple, either nestled in a town or secluded in a sacred grove. There you performed rites in honor of a god or pantheon. You served under a priest and studied religion. Thanks to your priest’s instruction and your own devotion, you also learned how to channel a modicum of divine power in service to your place of worship and the people who prayed there.</p>
+		</description>
+		<rules>
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SKILL_INSIGHT" />
+			<grant type="Proficiency" id="ID_WOTC_PHB24_PROFICIENCY_SKILL_RELIGION" />  
+			<grant type="Grants" id="ID_INTERNAL_GRANTS_BACKGROUND_ASI" />
+			<grant type="Feat" id="ID_WOTC_PHB24_FEAT_MAGIC_INITIATE" />
```

Comment:
No need to give them the "top level selectable feat". Just grant the Feat Feature.
```suggestion
			<grant type="Feat Feature" id="ID_WOTC_PHB24_FEAT_MAGIC_INITIATE_CLERIC" />
```